doc: Format of pcr primer JSON and v2 csv equivalent (#1389)
nextstrain/nextclade: docs/user/input-files/05-pathogen-config.md

#### `primers`

PCR primers to be used for primer mutation analysis. If not provided, no primer analysis is performed.

Example:

```json
// Below is the v3 representation of the following v2 input-primers-csv:
// Country (Institute),Target,Oligonucleotide,Sequence
// "Institut Pasteur, Paris (France)",RdRp,Pasteur_IP4_R,CTKGTCAAGGTTAATATAGG
"primers": [ // optional key, if not provided, no primer analysis is performed
  {
    "source": "Institut Pasteur, Paris (France)",
    "target": "RdRp",

>     "name": "Pasteur_IP4_R",                  // must be unique across all primers

Author: ```suggestion
    "name": "Pasteur_IP4_R",                  // must be unique for each primer
```

>     "rootOligonuc": "CCTATATTAACCTTGACCAG",   // substring of reference the primer binds to

Author: This should be `"refOligonuc"` not `"rootOligonuc"` - then it works
